joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/DependencyAstCreationPassTest.scala

```diff
@@ -168,6 +167,35 @@ class DependencyAstCreationPassTest extends AbstractPassTest {
        |import defaultMember2 from "module-name";
        |import "module-name";
        |""".stripMargin) { cpg =>
+      cpg.local.code.l shouldBe List(
+        "name",
+        "otherName",
+        "member1",
+        "alias1",
+        "member3",
+        "member4",
+        "member5",
+        "alias2",
+        "defaultMember1",
+        "alias3",
+        "defaultMember2",
+        "module-name"
+      )
+      cpg.assignment.code.l shouldBe List(
+        "var name = require(\"module-name\")",
+        "var otherName = require(\"module-name\")",
+        "var member1 = require(\"module-name\")",
```

Comment:
we can't figure out from such a call that `member1` refers to `^module-name^.member1`. It would have to be

```javascript
var member1 = require(\"module-name\").member1
```

Reply:
I see, let me fix that.

Reply:
now we have too many field accesses. We only want them for what MDN calls `exportN` at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#syntax

Reply:
Yes, done. I read over that section of the docs too quickly.